docs/installation.rst

```diff
@@ -71,11 +45,26 @@ is made to support Python 2 because it is considered obsolete as of early 2020.
 Testing your installation
 -------------------------
 
-``GMSO`` uses ``py.test`` to execute its unit tests. To run them, first install some extra depdencies:
+``GMSO`` uses ``py.test`` to execute its unit tests. To run them, first install the dev environment
 ::
-    $ conda install --file requirements-test.txt
 
+    $ git clone https://github.com/mosdef-hub/gmso.git
+    $ cd gmso
+    $ conda env create -f environment-dev.yml
+    $ conda activate gmso-dev
+    $ pip install -e .
```

Comment:
I'm not sure we need to repeat these steps. Instead we might reference the section header.